css/css-typed-om/the-stylepropertymap/properties/logical.html

```diff
@@ -23,5 +23,32 @@
   }
 }
 
+// borders
+for (const prefix of ['width', 'style']) {
+  for (const suffix of ['block', 'block-start', 'block-end', 'inline', 'inline-start', 'inline-end']) {
+    runPropertyTests('border-' + suffix + '-' + prefix, [
+      // TODO: Test 'auto'
+      { syntax: '<percentage>' },
+      { syntax: '<length>' },
+    ]);
+  }
+}
+
+for (const prefix of ['color']) {
```

Comment:
Can you also test border widths?
Border styles are keywords so I guess they can't be tested, given the `auto` TODO above?

Reply:
Line 27 above includes width and verifies it can take a length and percentage 👍 
 
![image](https://user-images.githubusercontent.com/1134620/91905844-01d36900-ec5c-11ea-99f8-1facb939b3fb.png)

border-style is keywords yeah, i think i can figure out how to test them. will update!

Reply:
Ah sorry, somehow I thought line 27 was for border shorthands like `border-block-start`. Not sure if it's possible to specify a syntax to test these

Reply:
give it all another look 👍